packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -479,6 +490,16 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
     });
   }
 
+  /* @conditional-compile-remove(unsupported-browser) */
+  public async populateEnvironmentInfo(): Promise<EnvironmentInfo> {
```

Comment:
actually given we call this in the adapter ctor.. can we make this private?

Reply:
(or perhaps not private, but remove from the CallAdapter interface and just keep on the AzureCommunicationCallAdapter)

Reply:
if it isn't on the interface we can't publicly export the handler from the AzureCommunicationsCallWithChatAdapter since it is viewing the adapter as a implemented CallAdapter. should we just remove the function entirely? this would stop Contoso from being able to ask for the adapter to get this data. (even though we ask for it in the constructor)